docs/getting-started/saas_tutorial.md

```diff
@@ -0,0 +1,376 @@
+
+<details>
+  <summary>Mac</summary>
+
+  ```sh
+  brew install sqlite
+  ```
+</details>
+
+<details>
+  <summary>Linux</summary>
+  <h4>Ubuntu/Debian-based systems</h4>
+
+  ```sh
+  sudo apt-get install sqlite3
+  ```
+
+  <h4>Fedora</h4>
+
+  ```sh
+  sudo dnf install sqlite
+  ```
+  <h4>CentOS/Redhat 7</h4>
+
+  ```sh
+  sudo yum install sqlite
+  ```
+
+  <h4>CentOS/Redhat 8+</h4>
+
+  ```sh
+  sudo dnf install sqlite
+  ```
+</details>
+
+### Installing Docker
+
+Follow the [instructions here](https://www.docker.com/get-started/) to install docker on your system.
+
+### Install the Mycelial CLI
+
+Follow the below instructions to install the Mycelial CLI:
+
+<details>
+  <summary>Mac</summary>
+
+  ```sh
+  brew install mycelial/tap/mycelial
+  ```
+
+</details>
+
+<details>
+  <summary>Linux</summary>
+
+  <h4>Debian Based Linux x86_64</h4>
+
+  ```sh
+  curl -L https://github.com/mycelial/cli/releases/latest/download/mycelial-v-1.x86_64.deb --output mycelial_amd64.deb
+  dpkg -i ./mycelial_amd64.deb
+  ```
+
+  <h4>Debian Based Linux ARM64</h4>
+
+  ```sh
+  curl -L https://github.com/mycelial/cli/releases/latest/download/mycelial-v-1.arm64.deb --output mycelial_arm64.deb
+  dpkg -i ./mycelial_arm64.deb
+  ```
+
+  <h4>Debian Based Linux ARM</h4>
+
+  ```sh
+  curl -L https://github.com/mycelial/cli/releases/latest/download/mycelial-v-1.armhf.deb --output mycelial_armhf.deb
+  dpkg -i ./mycelial_armhf.deb
+  ```
+
+  <h4>Redhat Based Linux x86_64</h4>
+
+  ```sh
+  yum install https://github.com/mycelial/cli/releases/latest/download/mycelial-v-1.x86_64.rpm 
+  ```
+
+  <h4>Redhat Based Linux ARM64</h4>
+
+  ```sh
+  yum install https://github.com/mycelial/cli/releases/latest/download/mycelial-v-1.arm64.rpm 
+  ```
+
+  <h4>Redhat Based Linux ARM</h4>
+
+  ```sh
+  yum install https://github.com/mycelial/cli/releases/latest/download/mycelial-v-1.armhf.rpm
+  ```
+
+
+</details>
+
+## Create a demo directory
+
+```sh
+mkdir mycelial-demo
+cd mycelial-demo
+```
+
+## Creating the source SQLite database
+
+In the command line, open sqlite:
+
+```sh
+sqlite3 source.db
+```
+
+Create a new users table with the following command:
+
+```sql
+CREATE TABLE users(id integer primary key, name text) STRICT;
+```
+
+Insert a new row in the `users` table:
+
+```sql
+INSERT INTO users(name) VALUES ('James');
+```
+
+Exit the SQLite command line program:
+
+```sql
+.exit
+```
+
+## Setup Postgres
+
+Start Postgres in a docker container with the following command:
+
+```sh
+docker run --name postgres-db -e POSTGRES_PASSWORD=secret -p 5432:5432 -d postgres
+```
+
+Enter the container and run the `psql` application with the following command:
+
+```sh
+docker exec -it postgres-db psql -U postgres
+```
+
+Create a new database with the following command:
+
+```sh
+CREATE DATABASE tutorial;
+```
+
+Exit out of the `psql` application with the following command:
+
+```sh
+\q
+```
+
+## Create an Account in the Mycelial App
```

Comment:
can we pick something other than "app"? like can we call it Mycelial Cloud or or maybe we just say "Create a Mycelial Account"